pages/api/_utils.ts

```diff
@@ -0,0 +1,71 @@
+import { ConfigHelper, Config, LoggerInstance } from '@oceanprotocol/lib'
+import Web3 from 'web3'
```

Comment:
As I have written in the original PR on market, if we can, we should rather use ethers.js. Then we do not need to use this old ens.js library which is due to be replaced with a new major version. In market we can't just use another web3 library, but here we have full flexibility about which libraries to use.

Ethers.js has multiple methods included which I tried to replicate with web3.js. So look into ethers.js and see what code we can ditch if we use that https://docs.ethers.io/v5/api/providers/provider/#Provider--ens-methods

Reply:
I've replaced it with ethers which seems to work nicely